### PR TITLE
Production recommendations document

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: docs/getting_started/production.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Cluster spec
+
+The `kops` command allows you to configure some aspects of your cluster, but for almost any production cluster, you want to change settings that is not accecible through CLI. The cluster spec can be exported as a yaml file and checked into version control.
+
+Read through the [cluster spec page](../cluster_spec.md) and familiarize yourself with the key options that kops offers.
+
+## Templating
+
+If your cluster contains multiple Instance Groups, or if you manage multiple clusters, you want to use generate the cluster spec using templates.
```
Comment (joshbranham) [inline docs/getting_started/production.md:31]:
```suggestion
If your cluster contains multiple Instance Groups, or if you manage multiple clusters, you want to generate the cluster spec using templates.
```

**Review by joshbranham — APPROVED** [review]
This is great 👍  thank you

**Comment by joshbranham** [comment]
Unsure about the netlify errors, but otherwise

/lgtm

**Comment by olemarkus** [comment]
Not sure what that is about either. `mkdocs build` doesn't work, but `import hack.mkdocs_macros` works ... `make live-docs` also works just fine.

**Comment by hakman** [comment]
I think a rebase should fix the mkdocs issue now that https://github.com/kubernetes/kops/pull/9988 was merged.

**Pull request merged**